src/main.ts

```diff
@@ -164,12 +164,13 @@ export function activate(context: vscode.ExtensionContext): ReturnType<typeof ge
             extension.logger.addLogMessage(`onDidSaveTextDocument triggered: ${e.uri.toString(true)}`)
             extension.manager.updateCachedContent(e)
             extension.linter.lintRootFileIfEnabled()
-            extension.structureViewer.update()
+            // Make sure to update cacheContent before calling structureViewer.update
```

Comment:
> before calling structureViewer.update

Change `update` to `computeTreeStructure`

Reply:
VS Code rename feature does not act on comments 😂